src/mono/sample/wasm/simple-server/Program.cs

```diff
@@ -19,6 +20,7 @@ public sealed class Program
     {
         private bool Verbose = false;
         private ConcurrentDictionary<string, Session> Sessions = new ConcurrentDictionary<string, Session>();
+        private Dictionary<string, byte[]> cache = new Dictionary<string, byte[]>();
```

Comment:
Should we use a case insensitive cache since file accesses would be case insensitive on Windows?